### DISCUSSION TITLE
Contact

### DISCUSSION THREAD

**Discussion by zeddosx** [Woodi-dev/Among-Us-Sheriff-Mod#37] [Q&A]
Hey man,
do you have an email I could contact you at? 
Thanks.

**Comment by Woodi-dev** [comment]
Woodi-dev@gmx.de

**Reply by ArielV12345** [reply]
Hey what up Woodi I heard you made some other exciting mods. I love the mods you make especially the mods on youtube that I watch. I was just wondering can I get a lend and try on these mods they seem fun and awesome. If so thank you and I will shout out you to everyone on my streams:) Thank you again:)

**Reply by ArielV12345** [reply]
If we can maybe we can contact on gmail? Thanks from Ariel:)
